### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dehall**
Adds content to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+mCODE is an initiative intended to assemble a core set of structured data elements for oncology electronic health records (EHRs). mCODE is a step towards capturing research-quality data from the treatment of all cancer patients. This would enable the treatment of every cancer patient to contribute to comparative effectiveness analysis of cancer treatments by allowing for easier methods of data exchange between health systems. mCODE has been created and is being supported by The American Society of Clinical Oncology (ASCO®) in collaboration with the MITRE Corporation. Read more at https://mcodeinitiative.org/ .
+
+## Camino
+Camino is a prototype SMART on FHIR application intended to demonstrate the value of mCODE through computable oncology clinical pathways. ASCO defines oncology clinical pathways as "detailed, evidence-based treatment protocols for delivering cancer care to patients with specific disease types and stages. When properly designed and implemented, oncology pathways can serve as an important tool in improving care quality and reducing costs." [(reference)](https://www.asco.org/practice-policy/cancer-care-initiatives/clinical-pathways)
```
Comment (radamson) [inline README.md]: `(reference)` looks odd in the readme.  Should the hyperlink text say `ASCO Clinical Pathways`, or use superscript and then add this as a footnote, or something else?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+![camino-logo-light](https://user-images.githubusercontent.com/13512036/73880187-ec17d580-482b-11ea-9b4f-e5f15a492fad.png)
+# Camino: mCODE™-based Oncology Clinical Pathways Prototype
+[![Build Status](https://travis-ci.org/mcode/pathways.svg?branch=master)](https://travis-ci.org/mcode/pathways)
 
-mCODE-based Oncology Clinical Pathways Prototype
+
+## mCODE: Minimal Common Oncology Data Elements
+mCODE is an initiative intended to assemble a core set of structured data elements for oncology electronic health records (EHRs). mCODE is a step towards capturing research-quality data from the treatment of all cancer patients. This would enable the treatment of every cancer patient to contribute to comparative effectiveness analysis of cancer treatments by allowing for easier methods of data exchange between health systems. mCODE has been created and is being supported by The American Society of Clinical Oncology (ASCO®) in collaboration with the MITRE Corporation. Read more at https://mcodeinitiative.org/ .
+
+## Camino
```
Comment (radamson) [inline README.md]: I feel like the Camino section should be above mCODE because that's what the repo is for.  Whatever works though.

**Review by radamson — APPROVED** [review]
Looks awesome.

Is it worth mentioning FHIR in the README somewhere?

**Review by danlee1025 — APPROVED** [review]
Looks great!

**Pull request merged**